File: API/Angular/src/app/viewmodels/UserRoundViewModels/response-user-round-view-model.ts
import { RoundStatus } from 'src/app/shared/enums/round-status';

export class ResponseUserRoundViewModel
{
  constructor(roundStatus, userId, nickname, points)
  {
    this.userId = userId;
    this.roundStatus = roundStatus;
    this.nickname = nickname;
    this.points = points;
  }

  public userId: number;
  public roundStatus: RoundStatus;
  public nickname: string;
  public points: number;
}

File: ViewModelsLayer/ViewModels/UserRoundViewModels/ResponseUserRoundViewModel.cs
using EntitiesLayer.Enums;

namespace ViewModelsLayer.ViewModels.UserRoundViewModels
{
    public class ResponseUserRoundViewModel
    {
        public int UserId { get; set; }
        public string Nickname { get; set; }
        public RoundStatusType RoundStatus { get; set; }
        public int Points { get; set; }
    }
}
